packages/wallets/src/styles/devices.scss

```diff
@@ -0,0 +1,24 @@
+/**
+ * Define Breakpoints here.
+ */
+$mobile-width: 320px;
+$tablet-width: 768px;
+$desktop-width: 1024px;
+
+@mixin desktop {
```

Comment:
we need to define the behaviour for sizes less than 320

Reply:
Can it be considered as mobile also? or we need another mixin?

Reply:
If you check the `320px ` in the code, you can find the exceptions from mobile view, but there is no specific design for wallets in this case, so it can be skipped for now and be considered later on.